src/Microsoft.ML.FastTree/RegressionTree.cs

```diff
@@ -63,14 +63,14 @@ public abstract class RegressionTreeBase
         /// bitwise complement operator in C#; for details, see
         /// https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/bitwise-complement-operator.
         /// </summary>
-        public IReadOnlyList<int> LteChild => _lteChild;
+        public IReadOnlyList<int> LessThanOrEqualToThresholdChildren => _lteChild;
```

Comment:
>LessThanOrEqualToThresholdChildren [](start = 34, length = 34)

It looks like across lightgbm, catboost and whatever they use in scikit this one called: leftChildrends, or leftChild, another one is rightChild.

Reply:
Will do leftChild for this.

---
In reply to: [261814959](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261814959) [](ancestors = 261814959)